CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateRecorder.swift

```diff
@@ -94,7 +94,14 @@ public class CRFHeartRateRecorder : RSDSampleRecorder, CRFHeartRateVideoProcesso
             else {
                 return nil
         }
-        let startTime = self.clock.startSystemUptime + 30
+        let timestampOffset: Double = {
+            guard let videoUptime = self._videoProcessor?.startSystemUptime
+                else {
+                    return 0.0
+            }
+            return videoUptime - self.clock.startSystemUptime
```

Comment:
Should this be: `self.clock.startSystemUptime - videoUptime` ?

Reply:
Yes, thanks. video startup is fast enough on my device that I didn't see this. Thanks!